packages/peregrine/lib/talons/ProductFullDetail/useProductFullDetail.js

```diff
@@ -315,10 +316,8 @@ export const useProductFullDetail = props => {
         handleAddToCart,
         handleSelectionChange,
         isAddToCartDisabled:
-            !isSupportedProductType ||
-            isMissingOptions ||
-            isAddConfigurableLoading ||
-            isAddSimpleLoading,
+            isMissingOptions || isAddConfigurableLoading || isAddSimpleLoading,
+        isSupportedProductType,
```

Comment:
Can list of supported product types be made a target in venia? Extensions can then add to this list.